homeassistant/components/ozw/websocket_api.py

```diff
@@ -21,13 +21,30 @@
 @callback
 def async_register_api(hass):
     """Register all of our api endpoints."""
+    websocket_api.async_register_command(hass, websocket_get_instances)
     websocket_api.async_register_command(hass, websocket_network_status)
     websocket_api.async_register_command(hass, websocket_node_metadata)
     websocket_api.async_register_command(hass, websocket_node_status)
     websocket_api.async_register_command(hass, websocket_node_statistics)
     websocket_api.async_register_command(hass, websocket_refresh_node_info)
 
 
+@websocket_api.websocket_command({vol.Required(TYPE): "ozw/get_instances"})
+def websocket_get_instances(hass, connection, msg):
+    """Get a list of OZW instances."""
+    manager = hass.data[DOMAIN][MANAGER]
+    instances = []
+
+    for instance in manager.collections["instance"]:
+        status = instance.get_status().data
```

Comment:
Is this returning a new copy of the data or the actual data? We're modifying it on the next row and we shouldn't modify the original data.

Reply:
Intent was to copy. I adjusted it to make sure we're returning a copy with the additional id field